mscore/exportxml.cpp

```diff
@@ -4729,6 +4729,58 @@ void ExportMusicXml::print(Measure* m, int idx, int staffCount, int staves)
 
       }
 
+//---------------------------------------------------------
+//  findAndExportTbdClef
```

Comment:
@lvinken could you please fix the comment? Should be `exportDefaultClef`.

Reply:
Done

Reply:
and squash the commits like `git rebase -i HEAD~2` and `git push -f origin 279671-default-clef-musicxml-export`?

Reply:
Also done